flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchApiCallBridge.java

```diff
@@ -40,12 +43,27 @@
 	/**
 	 * Creates an Elasticsearch {@link Client}.
 	 *
+	 * In comparison to {@link initClient}, this method creates a default {@link Client}.
+	 *
 	 * @param clientConfig The configuration to use when constructing the client.
 	 * @return The created client.
 	 */
 	Client createClient(Map<String, String> clientConfig);
```

Comment:
The `ElasticsearchApiCallBridge` actually isn't a user-facing class, only an interface for our version-specific implementations. So I think it is ok to alter the interface freely.

In that case, it shouldn't be necessary to have a new method.